### PR TITLE
blog: multisite replication with multisite replication wizard

### PR DESCRIPTION
**Pull request by aaSharma14**
This blog is for the new 4 Step wizard that is added to ceph dashboard Objects > Multi-site page

### CODE REVIEW

File: src/en/news/blog/2024/rgw-multisite-replication-wizard/index.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+### 4. Review and Token Generation
+
+The final step serves as a **review page** where you can verify all the values entered in the previous steps. If no additional cluster is added for replication, submitting this step generates a **token**. This token contains the realm name, access keys, and endpoints, and it can be manually imported into the secondary cluster using the `realm pull` command.
+
+However, if a secondary cluster is already present in the multi-cluster setup, you can select the cluster from the list, and the wizard will automatically import the realm token into the secondary cluster, completing the process seamlessly.
+
+![Step 4: Review and Token Generation](images/review-page.png)  
+
+
+**You can see a step by step progress on submition of the wizard.**
```
Comment (epuertat) [inline src/en/news/blog/2024/rgw-multisite-replication-wizard/index.md:64]: ?
```suggestion
**You can see a step by step progress on submitting of the wizard.**
```

File: src/en/news/blog/2024/rgw-multisite-replication-wizard/index.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+date: "2024-10-11"
+author: "Aashish Sharma"
+categories: "rgw"
+image: "images/multisite-replication-wizard.png"
+tags:
+  - "Ceph"
+  - "RGW multisite"
+  - "Multi-cluster management"
+  - "Object storage replication"
+  - "Data replication"
```
[the remark this answers is not in the document]
Comment (aaSharma14) [reply src/en/news/blog/2024/rgw-multisite-replication-wizard/index.md]: Updated, Thanks @epuertat

**Comment by nizamial09** [comment]
https://multisite-replication-wizard.ceph.io/en/news/blog/2024/rgw-multisite-replication-wizard/

**Comment by aaSharma14** [comment]
Hey @gauravsitlani, Can you please confirm who maintains this repo and whom should I contact to get this PR merged?

**Review by ljflores — APPROVED** [review]

**Comment by ljflores** [comment]
jenkins retest

**Pull request merged**